### PR TITLE
Add references (754, C, C++) for hexadecimal floating point format

### PR DESCRIPTION
**Pull request by cabo**

### CODE REVIEW

File: draft-ietf-cbor-edn-literals.md [hunk 1]
```diff
@@ -47,6 +47,37 @@ normative:
   IANA.core-parameters:
   BCP26: ianacons
   STD80: ascii
+  IEEE754:
+    target: https://ieeexplore.ieee.org/document/8766229
```
Comment (hildjj) [inline draft-ietf-cbor-edn-literals.md:51]: It infuriates me that this is correct. :(

File: draft-ietf-cbor-edn-literals.md [hunk 2]
```diff
@@ -764,7 +795,10 @@ The following additional items should help in the interpretation:
   ("0x"), base 8/octal ("0o"), or base 2/binary ("0b") notation, unless the
   optional part containing a "p" is present, in which case it stands
   for a floating point number in the usual hexadecimal notation (which
-  uses a mantissa in hexadecimal and an exponent in decimal notation).
+  uses a mantissa in hexadecimal and an exponent in decimal notation,
+  see Section 5.12.3 of {{IEEE754}}, Section 6.4.4.2 of {{C}}, or Section
+  5.3.14 of {{Cplusplus}}; floating-suffix/floating-point-suffix from
```
Comment (hildjj) [inline draft-ietf-cbor-edn-literals.md:800]: double check `5.3.14`?  I think you mean `5.13.4`.

**Review by hildjj — CHANGES_REQUESTED** [review]

File: draft-ietf-cbor-edn-literals.md (hunk 2 above)
Comment (cabo) [reply draft-ietf-cbor-edn-literals.md]: Indeed!  Fixed.

File: draft-ietf-cbor-edn-literals.md (hunk 1 above)
Comment (cabo) [reply draft-ietf-cbor-edn-literals.md:51]: I wish there were another "text available at" annotation here...
But at least IEEE is very open with institutional membership, so many engineers can get a login and free access through their employer.

**Review by hildjj — APPROVED** [review]

**Pull request merged**